airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -109,7 +113,11 @@ def check_query_status(self, query_execution_id: str) -> str | None:
         try:
             state = response["QueryExecution"]["Status"]["State"]
         except Exception as ex:
-            self.log.error("Exception while getting query state %s", ex)
+            self.log.error(
+                "Exception while getting query state %s. Query execution id: %s",
+                ex,
+                query_execution_id,
+            )
```

Comment:
Do we want to use `log.exception` instead? (Same for other calls below)

Reply:
I'm not sure if we should use `log.exception `.
In this PR I just added query execution id in the log message.

Reply:
@uranusjr I modify it to use `log.exception`